README.md

```diff
@@ -256,18 +256,20 @@ function renderHomepage() {
 
 Unlike the existing history API's `history.go()` method, which navigates by offset, navigating by key allows the application to not care about intermediate history entries; it just specifies its desired destination entry. There are also convenience methods, `appHistory.back()` and `appHistory.forward()`, and convenience booleans, `appHistory.canGoBack` and `appHistory.canGoForward`.
 
-All of these methods return promises, because navigations can be intercepted and made asynchronous by the `navigate` event handlers that we're about to describe in the next section. There are then several possible outcomes:
+All of these methods return `{ committed, finished }` pairs, where both values are promises. This because navigations can be intercepted and made asynchronous by the `navigate` event handlers that we're about to describe in the next section. There are then several possible outcomes:
 
-- The `navigate` event responds to the navigation using `event.transitionWhile()`, in which case the promise fulfills or rejects according to the promise(s) passed to `transitionWhile()`. (However, even if the promise rejects, `location.href` and `appHistory.current` will change.)
+- The `navigate` event cancels the navigation without responding to it, in which case both promises reject with an `"AbortError"` `DOMException`, and `location.href` and `appHistory.current` stay on their original value.
```

Comment:
By _cancels_, do you mean calling `event.preventDefault()`? or are there other ways?